### ISSUE TITLE
packages.dhall overrides message with example

### ISSUE THREAD

**Issue by waynevanson** [purescript/spago#734]
I'm unfamiliar with Dhall and I am having trouble understanding how to implement the following:

> "- if you know what you're doing and you want to disable this check, **you can override the `version` of the `metadata` package in the packages.dhall:**"
>
> ```dhall
> let overrides = { metadata = upstream.metadata // { version = "v0.14.0-rc5" } }
> ```
My current config is straight from `spago init`

```dhall
let upstream =
      https://github.com/purescript/package-sets/releases/download/psc-0.13.8-20201021/packages.dhall sha256:55ebdbda1bd6ede4d5307fbc1ef19988c80271b4225d833c8d6fb9b6fb1aa6d8

in  upstream
```

**Comment by waynevanson** [comment]
AH! So this is the solution:

```dhall
let upstream =
      https://github.com/purescript/package-sets/releases/download/psc-0.13.8-20201021/packages.dhall sha256:55ebdbda1bd6ede4d5307fbc1ef19988c80271b4225d833c8d6fb9b6fb1aa6d8

let overrides = { metadata = upstream.metadata // { version = "v0.14.0-rc5" } }

in  upstream // overrides
```

**Comment by waynevanson** [comment]
It might actually be worth adding this to the message if the message is directed to new users.

Use the projects `packages.dhall`and add the ` in upstream // overrrides` to the end of it too.

Would such a request be accepted?

**Comment by f-f** [comment]
@waynevanson fix up in #736 

The syntax in the error message and in the solution you found is correct, but recent versions have introduced a less verbose syntax, which we now suggest in the error message you got:
```dhall
let upstream =
      https://github.com/purescript/package-sets/releases/download/psc-0.13.8-20201021/packages.dhall sha256:55ebdbda1bd6ede4d5307fbc1ef19988c80271b4225d833c8d6fb9b6fb1aa6d8

in  upstream
  with metadata.version = "v0.14.0-rc5"
```
